src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -135,11 +141,17 @@ public void killConnection(String key) {
    * Requests a token from the key server.
    */
   public String requestTokenFromServer() {
+    //TODO: Replace when the StunFileGeneration is ready
+    pathToStunDumpFile = this.getClass().getClassLoader().getResource("ice4jDemoDump.ice")
```

Comment:
I would not use a a constant as it will be depending on where the program will be running. 
But yeah, its only temporary, so i moved it up, doesn't matter on my side.